src/index.js

```diff
@@ -7,9 +7,9 @@ import reportWebVitals from './reportWebVitals';
 const root = ReactDOM.createRoot(document.getElementById('root'));
 
 root.render(
-  <React.StrictMode>
+ 
```

Comment:
Keep in mind that most of the cases you want to keep react strict mode to make sure everything works as spected.